scriptEasy.js

```diff
@@ -10,10 +10,25 @@ const settings = {
   modal: document.querySelector('.modal'),
 };
 
-const text = `ass anv anb dd ssa ert lor s-- abc abc fde qwe asw wasd wsad`;
+function generateRandomSequence() {
+  const characters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890';
```

Comment:
Add some punctual signs to the characters constant value, so the user can practice the whole set of possible characters.